qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VisitNoteSteps.java

```diff
@@ -29,9 +29,9 @@ public class VisitNoteSteps extends Steps {
 
     private static final String DIAGNOSIS_PRIMARY = "Cancer";
     private static final String DIAGNOSIS_SECONDARY = "Malaria";
+    private static final String DIAGNOSIS_SECONDARY_UPDATED = "Pneumonia";
```

Comment:
What is this change for?

Reply:
This is to cater for` a user edits the visit note` on line  110.

Reply:
But you are simply removing and adding back the same exact thing.

Reply:
i reverted it back once.

Reply:
I have not yet seen the revert that you are tokinabout!

Reply:
And how about this?